src/themes/shared/typography.js

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import type {FontTokensT, TypographyT} from '../types.js';
+
+export const fontTokens: FontTokensT = {
+  primaryFontFamily:
+    'system-ui, "Helvetica Neue", Helvetica, Arial, sans-serif',
+};
+
+// color constants
+export default (primitives: FontTokensT = fontTokens): TypographyT => ({
```

Comment:
Had some trouble understand the argument here similar to above— primitives vs tokens. Maybe there's a clearer way handle the typography and overrides?